lib/runners/results.rb

```diff
@@ -62,11 +62,9 @@ def add_issue(issue)
       end
 
       def filter_issues(changes)
-        result = self.class.new(guid: guid, analyzer: analyzer)
-        issues.each do |issue|
-          result.add_issue(issue) if changes.include?(issue)
+        issues.reject! do |issue|
```

Comment:
How about this? See http://ruby-doc.org/core-2.7.0/Array.html#//dash_ref_method%252Di%252Dfilter%252D21/Method/filter!/0

```suggestion
        issues.filter! do |issue|
```

Reply:
Oh, I mistakenly used `reject!`! I should have chosen `filter!`. Thank you!

Reply:
I replaced `reject!` with `filter!` and did git-commit(1) with `--amend`.

Reply:
Sorry, reject! is correct. I'm looking into the error on CI.

Reply:
`filer!` is correct...